### ISSUE TITLE
help

### ISSUE THREAD

**Issue by byjyjs** [Fernando2603/AzurLane#42]
Hello, I am a beginner in this field and don't know what to do next after downloading. Can you explain it to me in detail? I know this is a very stupid question. Thank you very much.

**Comment by Fernando2603** [comment]
Hi, thank you for asking.

First, I want to apologize for the slow response, super busy lately. I also want to reassure you that your question isn’t stupid at all. The issue is that my repository currently lacks a guide for beginners, as it’s primarily structured for advanced developers who are already familiar with API usage and project setups.

### About the Repository
This repository contains Azur Lane media, such as ship skins, backgrounds, BGMs, skill icons, and more. It’s not intended for general users looking for media like BGMs or voice lines. Instead, it’s designed as a resource for developers to fetch and use this data programmatically.

### Usage Guidance
If you’re downloading the entire repository or following the structure documentation (or using @azurstarshine's guide), those are aimed at advanced users who already understand the repository structure and know what they’re doing.

For most intended use cases, you don’t need to download the whole repository. Instead, you can simply fetch or download the JSON files provided and use them in your programming projects. This repository is somewhat similar to AzurAPI, but it focuses solely on providing raw JSON data. You’ll need to implement the API logic yourself to make use of these resources.

### If You’re a Beginner
If you’re just starting or aren’t sure how to proceed, I’d recommend:
1. Reviewing the JSON files available and experimenting with them in your programming language of choice.
2. Exploring beginner-friendly tutorials for working with JSON and APIs to build your understanding.
3. Exploring [AzurLaneTools/AzurLaneLuaScripts](https://github.com/AzurLaneTools/AzurLaneLuaScripts), this gonna give you a lot of insight how of Azur Lane work.

### [This](https://celestialsparkz.github.io/#/ship) is example from old project of mine, to understand what you can do with this repository resource combined with [AzurLaneData](https://[github.com/AzurLaneTools/AzurLaneData](https://github.com/AzurLaneTools/AzurLaneData)), note: it's gonna take a while to load since it's not optimized and for testing purpose project.

### Feel free to ask if you have more questions or need additional guidance, I’ll do my best to help!

**Comment by byjyjs** [comment]
This may sound a bit difficult, but I will work hard to learn and understand. Thank you very much for your answer

**Comment by byjyjs** [comment]
This may sound a bit difficult, but I will work hard to learn and understand. Thank you very much for your answer

**Comment by azurstarshine** [comment]
@byjyjs I think we could give you a more useful answer if we understood what you are trying to do. What files do you need, and what do you intend to use them for?

Also, since @Fernando2603 mentioned [AzurLaneLuaScripts](https://github.com/AzurLaneTools/AzurLaneLuaScripts), I'd like to point out that the code in there is invalid. The decompiler that generated it is broken. At least one problem it has is that it does not generate all the variable definitions. It inlines the value for some variables, and then you see the variable referenced later in the code even though there is no definition or initialization. That makes it very difficult to understand what it's doing in some cases.

**Comment by Fernando2603** [comment]
I agree with @azurstarshine. That’s my mistake, my suggestion might indeed be challenging for someone who is a true beginner. When I wrote it, I assumed you already had some knowledge of programming languages and a bit of analytical experience.

Analyzing AzurLaneLuaScripts is probably too hard for beginners, but once you're a lot more experienced, I’d recommend giving it a try. The code there is super chaotic, but if you can read it, you’ll gain a deep understanding of the game’s core mechanics and how everything is calculated or executed.

I know I’m not a great teacher, and that’s probably why this repository doesn’t have any newbie guide. Maybe @azurstarshine can help you out.

I appreciate the clarification and hope it helps.

**Comment by byjyjs** [comment]
I am only interested in the game itself. I have learned to extract painting from game files, but the process is very complicated. I hope to obtain game resources more conveniently through a certain project, that's all. In fact, I was not even clear about his role before I opened this project.
